crates/cdk/src/wallet/multi_mint_wallet.rs

```diff
@@ -41,13 +53,26 @@ impl MultiMintWallet {
     }
 
     /// Add wallet to MultiMintWallet
-    #[instrument(skip(self, wallet))]
-    pub async fn add_wallet(&self, wallet: Wallet) {
+    pub async fn add_wallet(
```

Comment:
Why don't we keep add wallet the way it is currently done and then add a new fn `create_wallet` that does this. This way if a user wants to create a custom wallet with a different seed or something that can still be done and added to the multimint wallet.

Reply:
Done: e8c024ba559cf6162f12c4602b7dc78c5552e704